apps/meteor/server/startup/migrations/v315.ts

```diff
@@ -0,0 +1,11 @@
+import { Settings } from '@rocket.chat/models';
+
+import { addMigration } from '../../lib/migrations';
+
+addMigration({
+	version: 313,
```

Comment:
need to pay attention during conflict resolutions 
```suggestion
	version: 315,
```

Reply:
Sorry! You are right